mne/epochs.py

```diff
@@ -1599,6 +1600,29 @@ def __repr__(self):
         class_name = 'Epochs' if class_name == 'BaseEpochs' else class_name
         return '<%s | %s>' % (class_name, s)
 
+    def _repr_html_(self):
+        if self.baseline is None:
+            baseline = 'off'
+        else:
+            baseline = tuple(['None' if b is None else ('%.3f' % b)
+                              for b in self.baseline])
+            baseline = '%s: %s sec' % baseline
+
+        if isinstance(self.event_id, dict):
+            events = ''
+            for k, v in sorted(self.event_id.items()):
+                events += '%s: %i<br>' % (k, sum(self.events[:, 2] == v))
+        elif isinstance(self.event_id, list):
+            events = ''
+            for k in self.event_id:
+                events += '%s: %i<br>' % (k, sum(self.events[:, 2] == k))
+        elif isinstance(self.event_id, int):
+            events = '%i: %i' % (self.event_id, len(self.events[:, 2]))
+        else:
+            events = None
```

Comment:
Do you think you could

- assign `sum(self.events[:, 2] == v)`, `sum(self.events[:, 2] == k)`, `len(self.events[:, 2])` to a variable
- use this variable and `k` in an f-string, i.e. `events = f'{k}: {num_of_events}'`
?

Reply:
Alright!